skeleton/webpack/webpack.config.js

```diff
@@ -278,20 +279,23 @@ module.exports = ({ production, server, extractCss, coverage, analyze, karma } =
       // @if feat.babel
       {
         test: /\.js$/i, loader: 'babel-loader', exclude: nodeModulesDir,
-        options: coverage ? { sourceMap: 'inline', plugins: ['istanbul'] } : {}
+        options: tests ? { sourceMap: 'inline', plugins: ['istanbul'] } : {}
       },
       // @endif
       // @if feat.typescript
-      { test: /\.ts$/, loader: "ts-loader", options: { reportFiles: [ srcDir+'/**/*.ts'] }, include: srcDir },
```

Comment:
I think you accidentally removed this line from master.

Reply:
Ok, saw your commit message for this one too. I guess the side effect is that you have to turn on typeRoots unconditionally, which is not idea.

Reply:
My earlier take on the typeRoots involving monorepo might be little bit premature, as I have not recently tried to create a monorepo for a aurelia project. I tried that that a year back, and there were massive problems with resolving/loading symlinked packages; the webpack config was crowded with `alias`es. If this is a blocker, I will try to create a lerna prototype, and report the the findings.

Reply:
I tried to create a [monorepo using au-cli and lerna](https://github.com/Sayan751/au-cli-lerna). I couldn't make it work. It is not really because of the type definition, which makes the problem I suspected earlier. But even after fixing the typeRoots, Aurelia has problems loading the custom elements coming from different plugins. I opened an [issue](https://github.com/aurelia/webpack-plugin/issues/145) before on similar problem. I think the problems related to loading the symlinked packages needs to corrected before starting with bootstrapping lerna monorepo projects with lerna. 

On the other hand, this typing issue with Jasmine is already opened here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14569. If type definition  of jasmine is got resolved, probably the whole `typeRoots` thing can be done away with.

Reply:
I also discovered another bug with limiting ts-loader include. I am running release-check on v1 candidate, where I need to modify it to `include: karma ? [srcDir, testDir] : srcDir` to fix webpack+karma setup.

Reply:
Yes, I faced that too. Otherwise the `*.spec.ts` files are never processed, and webpack throws error. I think not having the `include` for `ts-loader` is far easier solution.

Reply:
I see. I will retain the "include" for v1 release. Then we can work on your big refactoring after v1.